From 01fdbd4e7d7064d9454aa6b0ca59835b30dcca6e Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai@quantatw.com>
Date: Wed, 25 Sep 2024 23:36:54 +0800
Subject: [PATCH 3/6] configurations: Revise CX7 NIC card temperature sensor

Revise the CX7 NIC temperature sensor name for Catalina system.

Signed-off-by: Potin Lai <potin.lai@quantatw.com>
Change-Id: I7ac22c0fb872f091141f2d77ef97b9686ddfcbae
---
 configurations/cx7_ocp.json | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/configurations/cx7_ocp.json b/configurations/cx7_ocp.json
index 3f11519..5ffa735 100644
--- a/configurations/cx7_ocp.json
+++ b/configurations/cx7_ocp.json
@@ -3,7 +3,7 @@
         {
             "Address": "$address",
             "Bus": "$bus",
-            "Name": "OCP CX7 NIC FRU $bus",
+            "Name": "OCP CX7 NIC FRU $bus % 2",
             "Type": "EEPROM"
         },
         {
@@ -12,11 +12,12 @@
             "Labels": [
                 "temp2"
             ],
-            "Name": "NIC Temp $bus",
-            "Name1": "NIC Temp $bus",
+            "Name": "NIC$bus % 2 TEMP_C",
+            "Name1": "NIC$bus % 2 TEMP_C",
             "Thresholds": [
                 {
                     "Direction": "less than",
+                    "Hysteresis": 0.1,
                     "Index": 2,
                     "Name": "lower critical",
                     "Severity": 1,
@@ -24,6 +25,7 @@
                 },
                 {
                     "Direction": "greater than",
+                    "Hysteresis": 2.1,
                     "Index": 2,
                     "Name": "upper critical",
                     "Severity": 1,
@@ -33,7 +35,7 @@
             "Type": "TMP421"
         }
     ],
-    "Name": "OCP CX7 NIC $bus",
+    "Name": "OCP CX7 NIC $bus % 2",
     "Probe": "xyz.openbmc_project.FruDevice({'BOARD_PRODUCT_NAME': 'Nvidia ConnectX-7 OCP3.0'})",
     "Type": "Board",
     "xyz.openbmc_project.Inventory.Decorator.Asset": {
-- 
2.49.0

